src/rust/engine/src/rule_graph.rs

```diff
@@ -786,16 +747,15 @@ fn entry_with_deps_str(entry: &EntryWithDeps) -> String {
       ref rule,
       ref params,
     }) => format!("{} for {}", rule, params_str(params)),
-    &EntryWithDeps::Root(ref root) => format!(
-      "{} for {}",
-      root
-        .clause
-        .iter()
-        .map(|s| select_root_str(s))
-        .collect::<Vec<_>>()
-        .join(", "),
-      params_str(&root.params)
-    ),
+    &EntryWithDeps::Root(ref root) => {
+      // TODO: Consider dropping this (final) public use of the keyword "Select", while ensuring
+      // that error messages remain sufficiently grokkable.
```

Comment:
I'd arguable that they are less understandable because of `Select` being included. I started using V2 after `Select` was already dropped, so it's Greek to me.

Reply:
Yep. This just preserves the status quo.